mantidproject/mantid
# Remove unused class after deleting Direct C2E interface

**Description of work**

In #21095 the Direct Convert to Energy interface was removed but a file depending on the `ui_` header file was left around. The PR builds would not pick this up as they are incremental but the clean builds fell over.

**To test:**

Before merging this run `find qt -name 'ui_*.h' -delete` from the build directory and then rebuild. You should get an error like [master_clean-rehl7](http://builds.mantidproject.org/view/Master%20Pipeline/job/master_clean-rhel7/569/console)

*No issue number*

**Release Notes** 
*Does not need to be in the release notes.*

---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
